api.planx.uk/pnpm-lock.yaml

```diff
@@ -7906,7 +7906,7 @@ packages:
     dev: true
 
   file:shared/core-1.0.0.tgz:
-    resolution: {integrity: sha512-nCa6FB4SqTxM0IN5OlM8QYlVEQJlpEZ6DG0esoFzmRNBxeNAsMwYbOi5puH8oIUiyw3Sz8gl+Htpppa/u+P3YA==, tarball: file:shared/core-1.0.0.tgz}
+    resolution: {integrity: sha512-G2n/IbC7IEwKxMoE+aTHXwhTFGSvwNd9YydWGeOVy2vnArpKakBXFsJSj0KWbfkpaVn8ZraWpUH3jAVWfm0Jjg==, tarball: file:shared/core-1.0.0.tgz}
```

Comment:
API & Editor lock file changes and `.tgz` files are a result of running `pnpm distribute` for the first time since pulling down re-named planx-client. It still feels a little odd/wrong to me to have them checked in here, is it okay?

Reply:
Yeah I'm hitting this also on a lot of PRs - I _think_ we can add them to the `.gitignore` as they'll always get generated by the CI. Is this correct @builditben ?